### DISCUSSION TITLE
How to record historical maximum value?

### DISCUSSION THREAD

**Discussion by RWTHLHK** [idaholab/moose#23346] [Q&A General] [answered]
I use a custom AuxKernel to help calculate the strain energy in an uni-axial loading simulation. I also need to record the maximum historical strain energy of each node, but I have no idea how to do that. Could anyone please give me some advice?

**Comment by GiudGiud** [comment, marked as answer]
Use a ElementExtremeValue to get the max on every time step

then use a ParsedPostprocessor to perform the max operation between the previous value (saved in the parsed one) and the current max

Guillaume
